kernel/classes/ezcontentobjecttreenode.php

```diff
@@ -2906,6 +2906,11 @@ static function updateMainNodeID( $mainNodeID, $objectID, $version = false, $par
         }
 
         $db->commit();
+        eZAudit::writeAudit( 'main-node-update', array( 'Parent Node ID' => $parentMainNodeID,
+                                              'Main Node ID' => $mainNodeID,
```

Comment:
I am going to update the code and log the old main node id and also the old parent node id